### ISSUE TITLE
Support for Angular 13

### ISSUE THREAD

**Issue by shruti-ahuja** [mattlewis92/angular-calendar#1536]
**Is your feature request related to a problem? Please describe**

Yes, Due to changes in the packaging of Angular 13 package format. Libraries should publish Ivy native distributions now. The currently released version of this library doesn't seem to have changed the format to the new version yet:

```
Processing legacy "View Engine" libraries:
- angular-draggable-droppable [es2015/esm2015] (git+https://github.com/mattlewis92/angular-draggable-droppable.git)

- angular-resizable-element [es2015/esm2015] (git+https://github.com/mattlewis92/angular-resizable-element.git)

- angular-calendar [es2015/esm2015] (git+https://github.com/mattlewis92/angular-calendar.git)
Encourage the library authors to publish an Ivy distribution.

```
It seems to work fine, but publishing the new format shouldn't be a big change and increases build speed and remove the warning.

**Describe the solution you'd like**

Update to Angular 13

**Describe your use case for implementing this feature**

Update to Angular 13

**Additional context**

**Comment by zpydee** [comment]
I am looking for an angular calendar and also need to know if you're planning a V13 ivy compatible version.

**Comment by piotrroda** [comment]
@mattlewis92 I'm also interested in.

**Comment by mattlewis92** [comment]
Will try and get to this over the weekend, I will probably end up shipping APF v12 but with the `enableIvy: false` flag removed so we can still support angular 12 users 😄

**Comment by mattlewis92** [comment]
I just released `0.29.0` to npm which requires angular 12 or higher and ships in ivy partial compilation mode so that warning should now be gone! 🥳 

Please note there are some other (small!) breaking changes to take account of: https://github.com/mattlewis92/angular-calendar/blob/master/CHANGELOG.md#0290-2021-12-11
